apps/settings/src/components/Users/UserRow.vue

```diff
@@ -351,6 +346,9 @@ export default {
 				wipe: false,
 				manager: false,
 			},
+			editedDisplayName: this.user.displayname,
+			editedPassword: '',
+			editedMail: this.user.email,
```

Comment:
```suggestion
			editedMail: this.user.email ?? '',
```

Ensure the email is not null, to prevent this:
![image](https://github.com/nextcloud/server/assets/1855448/5a070eef-99c7-4077-9103-f22e21a27da5)